lib/coins/ada.js

```diff
@@ -0,0 +1,41 @@
+const b58 = require('b58')
+const cbor = require('cbor-sync')
+const CRC32 = require('crc-32'); 
+
+module.exports = {depositUrl, parseUrl}
+
+function parseUrl(network, address) {
+  console.log('DEBUG16: [%s] *%s*', network, address)
+
+  // Cardano in Byron era not recognize network type (main/test) by address prefixes like f.e. bitcoin
+  if (!validate(address)) return null
```

Comment:
Convention: Lets move `validate` above this `parseUrl` function, so it is not used sooner than defined.